patches/minecraft/net/minecraft/world/chunk/Chunk.java.patch

```diff
@@ -185,15 +185,32 @@
      }
  
      public void func_76623_d()
-@@ -871,6 +873,7 @@
+     {
++        // FORGE - Fix for MC-92916
++        final List<net.minecraft.entity.player.EntityPlayer> players = new java.util.ArrayList<net.minecraft.entity.player.EntityPlayer>();
++        for (final ClassInheritanceMultiMap<Entity> multimap : field_76645_j)
++        {
++            for(final net.minecraft.entity.player.EntityPlayer player : multimap.func_180215_b(net.minecraft.entity.player.EntityPlayer.class))
++            {
++                players.add(player);
++            }
++        }
++        for (final net.minecraft.entity.player.EntityPlayer player : players)
++        {
++            field_76637_e.func_72866_a(player, false);
++        }
```

Comment:
I think this patch can be much smaller:
```java
++        // FORGE - Fix for MC-92916
++        for (ClassInheritanceMultiMap<Entity> multimap : entityLists)
++            for (net.minecraft.entity.player.EntityPlayer player : multimap.getByClass(net.minecraft.entity.player.EntityPlayer.class))
++                world.updateEntityWithOptionalForce(player, false);
```

Reply:
@bs2609 Yes, it was done that way to prevent a CME.

@mezz I'll see what I can do.